src/pivotal-ui/components/iconography/iconography.scss

```diff
@@ -90,13 +90,31 @@ Don't use the typography size modifier classes (e.g. .title, .h1, .type-sm)
 ```
 */
 
+.fa-h1 {
+  font-size: $font-size-h1 !important; 
+  line-height: $headings-line-height;
```

Comment:
I wonder if the line height also needs important? Not a big deal because we can fix it later without a breaking change.